near-plugins-derive/tests/access_controllable.rs

```diff
@@ -1520,3 +1534,41 @@ async fn test_acl_revoke_role_unchecked_is_private() -> anyhow::Result<()> {
     assert_private_method_failure(res, "acl_revoke_role_unchecked");
     Ok(())
 }
+
+const WASM_V_0_2_0_FILEPATH: &str = "tests/data/access_controllable_v_0_2_0.wasm";
+
+#[tokio::test]
+async fn test_upgrade_storage() -> anyhow::Result<()> {
+    let role = "ByMax2Increaser";
+
+    let old_wasm = std::fs::read(WASM_V_0_2_0_FILEPATH)?;
+    let Setup {
+        contract, account, ..
+    } = Setup::new_with_wasm(old_wasm).await?;
+
+    let contract_account = contract.contract().as_account();
+    let _ = contract
+        .acl_grant_role_unchecked(contract_account, role, account.id())
```

Comment:
Let's add more storage changes

Reply:
Added some more

Reply:
Can you please also add checks for these roles/changes

Reply:
Done